lib/puppet/transport/Schema/philips_hue.rb

```diff
@@ -0,0 +1,17 @@
+require 'puppet/resource_api'
+
+Puppet::ResourceApi.register_transport(
+  name: 'philips_hue',
+  desc: "Allow's manipulation of a philips hue light setup",
+
+  connection_info: {
```

Comment:
Potentially have a port option, so non standard ports can be used?

Reply:
Port option added